packages/core/src/codewhispererChat/commands/registerCommands.ts

```diff
@@ -69,3 +71,8 @@ export interface EditorContextCommand {
     type: EditorContextCommandType
     triggerType: EditorContextCommandTriggerType
 }
+
+export interface EditorContextCommandWithIssue extends EditorContextCommand {
+    type: 'aws.amazonq.fixCode'
+    codeScanIssue: CodeScanIssue
+}
```

Comment:
why extend an existing command instead of just defining a new one?

Reply:
why not?